docker-compose.yml

```diff
@@ -59,13 +60,17 @@ services:
             REDIS_URL: redis://redis:6379
             PYTHONFAULTHANDLER: 1
             PYTHONUNBUFFERED: 1
+            PYTHONDONTWRITEBYTECODE: 1
```

Comment:
This ensures that pyc files are not being written in development, so that changes to py files on the host are immediately represented in the running container.